tests/model_registry/rest_api/conftest.py

```diff
@@ -47,3 +47,35 @@ def updated_model_artifact(
         headers=model_registry_rest_headers,
         data_json=request.param,
     )
+
+
+@pytest.fixture()
+def updated_model_version(
+    request: pytest.FixtureRequest,
+    model_registry_rest_url: str,
+    model_registry_rest_headers: dict[str, str],
+    registered_model_rest_api: dict[str, Any],
+) -> dict[str, Any]:
+    model_version_id = registered_model_rest_api["model_version"]["id"]
+    assert model_version_id, f"Model version id not found: {registered_model_rest_api['model_version']}"
+    return execute_model_registry_patch_command(
+        url=f"{model_registry_rest_url}{MODEL_REGISTRY_BASE_URI}model_versions/{model_version_id}",
+        headers=model_registry_rest_headers,
+        data_json=request.param,
+    )
+
+
+@pytest.fixture()
+def updated_registered_model(
+    request: pytest.FixtureRequest,
+    model_registry_rest_url: str,
+    model_registry_rest_headers: dict[str, str],
+    registered_model_rest_api: dict[str, Any],
+) -> dict[str, Any]:
+    model_id = registered_model_rest_api["register_model"]["id"]
+    assert model_id, f"Model id not found: {registered_model_rest_api['register_model']}"
+    return execute_model_registry_patch_command(
+        url=f"{model_registry_rest_url}{MODEL_REGISTRY_BASE_URI}registered_models/{model_id}",
+        headers=model_registry_rest_headers,
+        data_json=request.param,
+    )
```

Comment:
These are the same fixture basically, the only thing that is changing is the "operation". Could you do a single fixture that takes as a parameter the operation we want to do (model_version/register_model)?